auto_builder.sh

```diff
@@ -60,7 +60,8 @@ echo "     5 - Atftp"
 echo "     6 - Powerman"
 echo "     7 - Conman"
 echo "     8 - iPXE roms"
-echo " 9 - Exit."
+echo "     10 - bluebanquise"
+echo " 11 - Exit."
```

Comment:
9 is missing. Might be resolved after merge of #9?

Reply:
Yes.
Let me also add the Q for Quit.

Reply:
Ok done.